docs/usage/key-concepts/dependency-dashboard.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Dependency Dashboard
+description: Learn all about Renovate's Dependency Dashboard
+---
+
+# Introduction
+
+Renovate has a Dependency Dashboard that shows a overview of the state of your repositories' dependencies.
+
+When you turn on the Dependency Dashboard, Renovate will create a new issue on the repository.
+This issue contains a "dashboard" where you can get a overview of the status of all updates.
+
+Having the Dependency Dashboard enabled also allows you to opt-in to different behavior for certain or even all updates with the "Dependency Dashboard Approval" workflow.
+
+## Supported platforms
+
+The Dependency Dashboard requires that the host platforms supports the concept of issues.
+Read [our FAQ, Renovate core features not supported on all platforms](https://docs.renovatebot.com/faq/#renovate-core-features-not-supported-on-all-platforms) to see if your platform can use the Dependency Dashboard feature.
+
+## How to enable the dashboard
+
+<!-- TODO: It might be nice to change our config presets, so we have one to `:enableDependencyDashboard` and one to `:disableDependencyDashboard`. -->
+
+To turn on the Dashboard manually, add the `:dependencyDashboard` preset to your `extends` array in the Renovate configuration file:
+
+```json
+{
+  "extends": ["config:base", ":dependencyDashboard"]
+}
+```
+
+Or set `dependencyDashboard` to `true`:
+
+```
+{
+  "dependencyDashboard": true
+}
+```
+
+<!--
+TODO: discuss whether we need to change things.
+
+https://docs.renovatebot.com/configuration-options/#dependencydashboardapproval
+says to set `dependencyDashboardApproval` to `true`, but we also have the `:dependencyDashboard`, which is not mentioned in the link.
+
+-->
+
```

Comment:
```suggestion
```

Turning on any approvals implicitly enables the dashboard, so the documentation is correct